### PR TITLE
Added check in bulk reindex to skip deleted records

### PR DESCRIPTION
**Pull request by pazembrz**

### CODE REVIEW

File: inspirehep/modules/records/tasks.py [hunk 1]
```diff
@@ -190,6 +190,9 @@ def actions():
         for uuid in uuids:
             try:
                 record = InspireRecord.get_record(uuid)
+                if record.get('deleted', False):
+                    logger.debug("Record already %s deleted, not indexing!", uuid)
+                    continue
```
Comment (drjova) [inline inspirehep/modules/records/tasks.py]: can we add a test for it?

**Comment by pazembrz** [comment]
ci test please

**Comment by pazembrz** [comment]
ci test please

**Comment by pazembrz** [comment]
ci test please

**Comment by pazembrz** [comment]
ci test please

**Comment by pazembrz** [comment]
ci merge please

**Comment by drjova** [comment]
ci merge please

**Comment by drjova** [comment]
ci merge please

**Pull request merged**